tests/views/test_course_top_stats_metrics_view.py

```diff
@@ -0,0 +1,106 @@
+
+import mock
```

Comment:
Did you check all metrics tests are passing?

Reply:
No. Each metric has a separate file for tests. I followed pattern and created a new file for this API's tests. I don't think checking/fixing all metrics tasks is in the scope of this sub-task as well.

Reply:
if @zubair-arbi and @ziafazal think that we need to fix all the tests for figures for the MVP, there needs to be created a separate story for that because I don't think a lot of tests would pass as they are not written very well as well as they are written keeping in mind Appsembler's multistite architecture.

Reply:
@ziafazal @zubair-arbi as we are using many API with changes expected in results returned. I suggest we fix these unit tests. Kindly guide

Reply:
@asadmanzoor93 for MVP, we just need to update/fix the tests which we modify or add.